src/main/target/MATEKF405/target.h

```diff
@@ -176,6 +176,9 @@
 
 #define PITOT_I2C_BUS               DEFAULT_I2C_BUS
 
+#define USE_OPTICAL_FLOW
```

Comment:
`USE_OPTICAL_FLOW` is not a used macro, maybe you meant `USE_OPFLOW` ? In this case it is already defined for all targets having more than 256k flash, it is not needed.